ci/build/build-lib.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
```

Comment:
suggest using `set -euo pipefail` here, this also means `$ARCH` needs to be `${ARCH:-}` etc

Reply:
AHHH! I had that and it kept failing and I didn't know why. Thanks for the missing piece of the puzzle 🙌